Tests/Editor/Bootstrap/BootstrapSettingsTests.cs

```diff
@@ -421,141 +370,144 @@ public void CurrentRegion_WhenRefreshBucketNameAndRegionAndAccountIdAndProfileSa
         {
-            const string testBucketName = "test-bucket";
-            const string testRegion = "test-region";
-
-            var coreApiMock = new Mock<CoreApi>();
-            coreApiMock.SetUpCoreApiWithRegion(success: true, valid: true, testRegion: testRegion);
-            coreApiMock.SetUpCoreApiWithAccountId(success: true);
-            coreApiMock.SetUpCoreApiWithProfile(success: true);
-
             var bucketNameFormatterMock = new Mock<IBucketNameFormatter>();
-            bucketNameFormatterMock.Setup(target => target.FormatBucketName(It.IsAny<string>(), testRegion))
-                .Returns(testBucketName)
+            bucketNameFormatterMock.Setup(target => target.FormatBucketName(It.IsAny<string>(), TestRegion))
+                .Returns(TestBucketName)
                 .Verifiable();
 
-            BootstrapSettings underTest = GetUnitUnderTest(bucketNameFormatterMock, coreApiMock);
+            BootstrapSettings underTest = GetUnitUnderTest(bucketNameFormatterMock);
 
             // Act
             underTest.RefreshBucketName();
             string name = underTest.BucketName;
 
             // Assert
-            coreApiMock.Verify();
-            Assert.AreEqual(testBucketName, name);
+            _coreApiMock.Verify();
+            Assert.AreEqual(TestBucketName, name);
         }
 
         [Test]
         public void BucketName_WhenNewInstance_IsNull()
         {
             var bucketNameFormatterMock = new Mock<IBucketNameFormatter>();
-
             BootstrapSettings underTest = GetUnitUnderTest(bucketNameFormatterMock);
             Assert.IsNull(underTest.BucketName);
         }
 
         private static BootstrapSettings GetUnitUnderTest(Mock<IBucketNameFormatter> bucketNameFormatterMock = null,
-            Mock<CoreApi> coreApi = null,
             Mock<BootstrapUtility> bootstrapUtility = null)
         {
             bucketNameFormatterMock = bucketNameFormatterMock ?? new Mock<IBucketNameFormatter>();
-            coreApi = coreApi ?? new Mock<CoreApi>();
-            bootstrapUtility = bootstrapUtility ?? new Mock<BootstrapUtility>(coreApi.Object);
-            TextProvider textProvider = TextProviderFactory.Create();
+            bootstrapUtility = bootstrapUtility ?? new Mock<BootstrapUtility>(_coreApiMock.Object);
 
+            TextProvider textProvider = TextProviderFactory.Create();
             string[] policyNames = new string[] { "none", "30 days" };
             var lifecyclePolicies = new BucketPolicy[] { BucketPolicy.None, BucketPolicy.ThirtyDaysLifecycle };
-            return new BootstrapSettings(lifecyclePolicies, policyNames, textProvider, bucketNameFormatterMock.Object, new MockLogger(), coreApi.Object, bootstrapUtility.Object);
+            return new BootstrapSettings(lifecyclePolicies, policyNames, textProvider, bucketNameFormatterMock.Object, new MockLogger(), _stateManager.Object, _coreApiMock.Object, bootstrapUtility.Object);
```

Comment:
It looks like you misunderstood what I meant in my first review of this PR, because this change appears to be dependent on the file changes you moved to [PR 118](https://github.com/aws/amazon-gamelift-plugin-unity/pull/118/files#diff-e07322110baf76102ea9f1b74868a21043364e23fb83d0e0425e11414163c76d).

You should never be submitting PRs that don't build. Our assumption is that you've built the PR and the tests have passed. We are reviewing the code, we are not downloading each revision of your PR and building it ourselves to check the tests pass.

I understand that this has not been the case so there were already failing tests. That said you should keep your PRs concise and relevant. For example, a PR that just fixes broken out of date tests missed in a previous PR, a PR that adds new functionality (like BootstrapSettings using StateManager you have in [PR 118](https://github.com/aws/amazon-gamelift-plugin-unity/pull/118/files#diff-e07322110baf76102ea9f1b74868a21043364e23fb83d0e0425e11414163c76d)) as well as it's updated unit tests (this change), then a PR that updates namespaces which should just be clean up and shouldn't handle functionality at all.

Reply:
Should be all split up now into correct PRs